crates/subspace-node/src/chain_spec.rs

```diff
@@ -380,6 +382,19 @@ fn subspace_genesis_config(
     } = genesis_params;
 
     let genesis_domains = if enable_domains {
+        let bundle_slot_probability = (1, 1);
+
+        let DomainBundleLimit {
+            max_bundle_size,
+            max_bundle_weight,
+        } = calculate_max_bundle_weight_and_size(
```

Comment:
Instead of doing this calculation here, I'd imagine this would be how a default value for a new domain config is computed for any new domain even registered after genesis unless different numbers are provided

Reply:
Make sense! will do that